.github/workflows/bloat_check.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
         runs-on: ubuntu-latest
 
         container:
-            image: ghcr.io/project-chip/chip-build:32
+            image: ghcr.io/project-chip/chip-build:35
```

Comment:
What do these changes have to do with the Chef video player?  It looks like this PR has a whole bunch of unrelated changes in it...

Reply:
@bzbarsky-apple Thanks for reviewing this. I have some conflicts during rebasing so this PR contains other people's commits during rebasing. Please ignore the current one (I'll close it soon) and help to review #32091 instead.